utils.js

```diff
@@ -4266,14 +4479,39 @@ export function repoMetadataToGitHubApiUrl(repoMetadata) {
     const name = repoMetadata.name;
     let ghUrl = "https://api.github.com/repos";
     if (group && group !== "." && group !== "") {
-      ghUrl = `${ghUrl}/${group.replace("github.com/", "")}`;
+      ghUrl = `$ghUrl/${group.replace("github.com/", "")}`;
```

Comment:
Can you revert this since looks incorrect.

Reply:
I'm a bit puzzled by how this change occurred. I don't recall making any modifications to this particular function.